### PR TITLE
Add the missing HOST_TRANSACTION_REQUESTS_SERVICE variable

### PR DESCRIPTION
**Pull request by lewisdaly**
I wasn't sure what this should be for the `Mojaloop-Local-Docker-Compose.postman_environment.json` file, but can someone please refer me to the right place and I can add it?

### CODE REVIEW

File: environments/Mojaloop-Local.postman_environment.json [hunk 1]
```diff
@@ -771,6 +771,11 @@
 			"key": "HOST_BULK_API_ADAPTER",
 			"value": "http://localhost:3003",
 			"enabled": true
+		},
+		{
+			"key": "HOST_TRANSACTION_REQUESTS_SERVICE",
+			"value": "http://transaction-request-service.local",
```
Comment (elnyry) [inline environments/Mojaloop-Local.postman_environment.json]: transaction-requests-service.local I think ('s' after request)

Comment (lewisdaly) [reply environments/Mojaloop-Local.postman_environment.json]: But in the helm ingress, it isn't for some reason.

https://github.com/mojaloop/helm/blob/master/transaction-requests-service/values.yaml#L88

**Comment by lewisdaly** [comment]
I'll also include a somewhat controversial change: a script to sort the environment variables alphabetically for us, as well as a reordering of the current environment variables. Happy to put in a separate PR if needed.

File: environments/Mojaloop-Local.postman_environment.json (hunk 1 above)
Comment (elnyry) [reply environments/Mojaloop-Local.postman_environment.json]: Yeah you're right, thats the value used in helm ingress..

**Comment by elnyry** [comment]
> I'll also include a somewhat controversial change: a script to sort the environment variables alphabetically for us, as well as a reordering of the current environment variables. Happy to put in a separate PR if needed.

yeah that's probably better..

**Comment by lewisdaly** [comment]
@elnyry I took out the ordering stuff as requested

**Pull request merged**